server/views/templates/explore-landing.njk

```diff
@@ -1,6 +1,8 @@
 {% extends "pages/explore-landing.njk" %}
 
 {% block body %}
+  {% component 'page-description', { title: 'Explore landing', modifiers: ['hidden'] } %}
```

Comment:
Ah, I forgot there was designs for this page... I was thinking about our beautiful list that we have now.